runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -175,9 +175,13 @@ public void onBeforeMenuRender(BeforeMenuRender event)
 	{
 		if (config.hdMenu())
 		{
-			client.draw2010Menu();
-			event.consume();
+			client.draw2010Menu(config.menuAlpha());
```

Comment:
It's not super-important, but this change should be in the first commit so the commits are atomic.